code/data_structures/src/tree/binary_tree/binary_tree/traversal/postorder/postorder.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+
+using namespace std;
```

Comment:
Also remove using directive to avoid naming conflict.

Reply:
Will take care of this from now :+1: